sandbox/public/index.html

```diff
@@ -29,8 +29,10 @@
         debug: true
       });
 
-      alloy("viewStart", {
-        type: "view:start",
+      alloy("event", {
+        // NOTE: `viewstart` is a special type of events.
+        // View docs for more info.
+        type: "viewstart",
```

Comment:
This is the reason why we had a special "viewstart" command in the first place.  Namely, we didn't want to have to dictate that this type parameter be passed here.  It would be more explicit if we had a flag to tell personalization to do its "special" logic for viewstart.  i.e. we could have a flag like `fetchPersonalization: true`.  What do you think?

Reply:
Is there a reason we wouldn't want to dictate that the `type` parameter be passed here? It seems like whether `viewstart` is indicated as a separate command, `type` option, or `fetchPersonalization` option, we're going to be dictating it regardless. The only thing I could see potentially wrong with using `type` is if a consumer would want to provide a unique `type` value when sending an event for a "view start"....but the purposes and usage of the `type` option is still bit vague to me.

Reply:
After talking with the edge team, product and few Alpha customers, it seems like `viewStart` might be the first of few `built-in` events that the Adobe solutions can leverage.

There maybe be some use cases where you might call `alloy("event"..` and not want to pass a `type` but I really can't think of any. If the visitor has triggered a specific event like maybe `logIn` or `addToCart`.. or if the Activity collector has triggered some click on a link or button.. or a passive event has occurred like a video finished playing.. you will always need a type for all those events.

If most of us agree, I'd like to have this change be part of the next Alpha-2 release and get feedback in the next Alpha meeting.

Reply:
👍 from me.